src/client/components/DistrictOptionsFlyout.tsx

```diff
@@ -18,44 +22,29 @@ const DistrictOptionsFlyout = ({
   readonly isDistrictHovered: boolean;
 }) => {
   return (
-    <Wrapper
-      onSelection={(menuKey: MenuKeys) => {
-        const action =
-          menuKey === "zoom-to-district" ? setZoomToDistrictId(districtId) : assertNever(menuKey);
-        store.dispatch(action);
-      }}
-      style={{ visibility: isDistrictHovered ? "visible" : "hidden" }}
-    >
-      <Tooltip content="More options">
-        <MenuButton
-          sx={{
-            ...{
-              variant: "icon",
-              fontWeight: "light",
-              px: 1,
-              py: 1,
-              color: "gray.8",
-              bg: "transparent",
-              "&:hover:not([disabled]):not(:active)": {
-                bg: "rgba(89,89,89,0.1)"
-              },
-              borderRadius: "100%"
-            }
-          }}
+    <Menu
+      viewScroll="close"
+      animation={false}
+      offsetX={30}
+      menuButton={
+        <Button
+          sx={{ mr: 2 }}
+          variant="icon"
+          style={{ visibility: isDistrictHovered ? "visible" : "hidden" }}
         >
-          ···
-        </MenuButton>
-      </Tooltip>
-      <Menu sx={{ ...style.menu, marginTop: "-35px", right: "-210px" }}>
-        <ul sx={style.menuList}>
-          <li>
-            <MenuItem value={"zoom-to-district"}>
-              <Box sx={style.menuListItem}>Zoom to District</Box>
-            </MenuItem>
-          </li>
-        </ul>
-      </Menu>
-    </Wrapper>
+          <Icon name="ellipsis" color="#131f28" size={0.75} />
+        </Button>
+      }
+    >
+      <MenuItem
+        styles={{
+          fontSize: "14px"
+        }}
+        value={"zoom-to-district"}
```

Comment:
This should make zoom to district work again:
```suggestion
        onClick={() => {
          store.dispatch(setZoomToDistrictId(districtId));
        }}
```